builder/xenserver/common/step_type_boot_command.go

```diff
@@ -68,13 +68,15 @@ func (self *StepTypeBootCommand) Run(ctx context.Context, state multistep.StateB
 	}
 
 	location, err := c.client.Console.GetLocation(c.session, consoles[0])
+	ui.Say(fmt.Sprintf("LOCATION: %s", location))
 	if err != nil {
 		ui.Error(err.Error())
 		return multistep.ActionHalt
 	}
-
+	locationPieces := strings.SplitAfter(location, "/")
+	consoleHost := strings.TrimSuffix(locationPieces[2], "/")
 	ui.Say("Connecting to the VM console VNC over xapi")
```

Comment:
I'd like to consolidate this message with the message on line 95. Printing out a log message with just the IP doesn't make it obvious as to what is happening.

How about this instead?

```
ui.Say(fmt.Sprintf("Connecting to the VM console VNC over xapi via %s", consoleHost))
```

Reply:
Totally agree with you, that was only for debugging purposes.